src/xunit.performance.api/XunitPerformanceHarness.cs

```diff
@@ -71,6 +75,66 @@ public void RunBenchmarks(string assemblyPath)
             s_profiler(assemblyPath, Configuration.RunId, _outputDirectory, runner, collectOutputFilesCallback);
         }
 
+        public void RunStartupBenchmark(string[] args, ProcessStartInfo psi, PreDel pre, MidDel mid, PostDel post)
+        {
+            pre(_performanceTestConfig, psi);
+            int iterations = _performanceTestConfig.iterations;
+            int timeout = _performanceTestConfig.timeout;
+            psi.RedirectStandardOutput = true;
```

Comment:
>psi.RedirectStandardOutput = true; [](start = 12, length = 34)

`psi` is user input parameter. I would not change this without letting the user know, or make it explicit on the function header/comment (it is a public API, and we should be transparent).